tools/ARIAtools/ARIAProduct.py

```diff
@@ -180,10 +180,9 @@ def __mappingData__(self, file, rmdkeys, sdskeys):
         datalyr_dict={}
 
         # Setup rdrmetadata_dict
-        # for i, j in enumerate(rdrmetakeys):
-        for i, j in enumerate(rdrmetakeys):
+        for j in enumerate(rdrmetakeys):
             try: #If layer expected
-                rdrmetadata_dict[radarkeys[rmdkeys.index(j)]]=rdrmetadata[j][0]
+                rdrmetadata_dict[radarkeys[rmdkeys.index(j[1])]]=rdrmetadata[j[1]][0]
```

Comment:
@sssangha please confirm

Reply:
@ehavazli ok, it looks like enumeration is not needed any longer. So why not just have a classic for loop? In that case, there isn't a need to catch the 0th index for each iterated object? I see this also a couple of other for loops in productPlot and extractProduct. Let me know if this makes sense, and I could go ahead and adjust it accordingly

Reply:
yes please go ahead, codacy was complaining about having two variables and looping around range(len()).